src/main/kotlin/com/atlassian/performance/tools/ssh/PerformanceDefaultConfig.kt

```diff
@@ -0,0 +1,25 @@
+package com.atlassian.performance.tools.ssh
+
+import net.schmizz.sshj.DefaultConfig
+import net.schmizz.sshj.transport.random.BouncyCastleRandom
+import net.schmizz.sshj.transport.random.JCERandom
+import net.schmizz.sshj.transport.random.Random
+import net.schmizz.sshj.transport.random.SingletonRandomFactory
+import net.schmizz.sshj.common.Factory
+
+internal class PerformanceDefaultConfig : DefaultConfig() {
+    companion object {
+        val bcFactory = MemoizingFactory(BouncyCastleRandom.Factory())
+        val jceFactory = MemoizingFactory(JCERandom.Factory())
+    }
+    override fun initRandomFactory(bouncyCastleRegistered: Boolean) {
```

Comment:
How did you find this was the problem? I'm awe-struck.

Reply:
We had the same case in Bamboo in 2013 - actually two cases, one with SCP plugin and the other one with private key generation for EC2 tunnel

Reply:
Experience FTW 😎